[quant] Remove get_qparams in Observers

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #38552 [quant] Make MinMaxObserver and MovingAverageMinMaxObserver work with List[Tensor]
* #38436 [quant] Histogram Observer support List[Tensor]
* #38667 [quant][graphmode] Different rule for add/add_/mul/mul_
* #38570 [quant][graphmode] Different rule for handling `aten::cat`
* **#38435 [quant] Remove get_qparams in Observers**

Differential Revision: [D21597835](https://our.internmc.facebook.com/intern/diff/D21597835)

jerryzh168 committed:
[quant] Remove get_qparams in Observers

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Remove get_qparams in Observers"




[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Remove get_qparams in Observers"




[ghstack-poisoned]

supriyar commented on torch/quantization/observer.py:
```diff
@@ -592,11 +580,6 @@ def _forward(self, x_orig):
     def calculate_qparams(self):
```
Do you need to call `torch.jit.export` on this method now?

jerryzh168 replied:
yeah, I think so, will add it

jerryzh168 committed:
Update on "[quant] Remove get_qparams in Observers"




[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Remove get_qparams in Observers"




[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Remove get_qparams in Observers"


Differential Revision: [D21597835](https://our.internmc.facebook.com/intern/diff/D21597835)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Remove get_qparams in Observers"


Differential Revision: [D21597835](https://our.internmc.facebook.com/intern/diff/D21597835)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Remove get_qparams in Observers"


Differential Revision: [D21597835](https://our.internmc.facebook.com/intern/diff/D21597835)

[ghstack-poisoned]
